azure-devops/projects/npm-packages/project.tf

```diff
@@ -0,0 +1,7 @@
+resource "azuredevops_project" "project" {
+  name               = "npm-packages"
+  description        = "This is the DevOps project for all public npm packages"
+  visibility         = "public"
+  version_control    = "Git"
+  work_item_template = "Basic"
+}
```

Comment:
copy from this https://github.com/pagopa/gitops/blob/main/azure-devops/projects/cgn-onboarding-portal-projects/project.tf